foundation/web/web.go

```diff
@@ -100,6 +100,7 @@ func (a *App) corsHandler(webHandler HandlerFunc) HandlerFunc {
 		for _, origin := range a.origins {
 			if origin == "*" || origin == reqOrigin {
 				w.Header().Set("Access-Control-Allow-Origin", origin)
+				break
```

Comment:
I was looking at this more. I do except a collection of origins. I'm not really up to speed on CORS so I am wondering why I ever did that? Maybe a.origins should not be a slice?

Reply:
Also, pull the latest code because I added some comments.